### ISSUE TITLE
Image enhancement

### ISSUE THREAD

**Issue by vokaliz** [rinigus/pure-maps#322]
In attachment screenshot, look pls. So the navigation appears on Meizu pro 5 with ubports ota-10. Can you make the information more readable in the upper left corner. 
Thank you for the cool navigation for ubuntu touch!
Best regards, Dema!

![screenshot20190829_194757407](https://user-images.githubusercontent.com/39481201/64061785-e1f00c00-cbe7-11e9-941c-828476c6c80c.png)

**Comment by rinigus** [comment]
That looks to be rather excessive translation in Russian of ETA. As a result, strings overlap. Any better suggestion? Currently we have 

ETA -> Расчетное время прибытия

**Comment by vokaliz** [comment]
And what if the ETA "Расчетное время прибытия“ is shifted below or if there is no such possible, then use the abbreviated phrase "Расч. время прибытия". The phrase is shorter and quite understandable in russian.

**Comment by rinigus** [comment]
any 3 char abbreviations? maybe something used in other apps? Its still rather long. In future that will move, but its planned to move it to the place where it is not expected to take much space either

**Comment by vokaliz** [comment]
If the phrase will be moved to another place, this will be the best solution. Thank you for answer!

**Comment by rinigus** [comment]
No, in the new place there is no more space either. At least, I was not planning to allocate too much space for it. So, ideally, short abbreviation is needed

**Comment by rinigus** [comment]
any idwas on shorter version of eta in Russian ?

**Comment by vokaliz** [comment]
1. For example: "Расч. время прибытия".
2. Or this one: "Расч. вр. прибытия".
3. And one more: "Расч-ное время приб-я".
For me number 1 is preferable.

**Comment by rinigus** [comment]
What about 'прибытие'?

**Comment by vokaliz** [comment]
We can cut this word like - "... приб." 
All phrase - "Расч. время приб".

**Comment by rinigus** [comment]
In this case, its clear that we are talking about time as the time is shown next to it. Looks like HERE is using exactly the same word as I suggested: https://a.d-cd.net/9b59726s-960.jpg

**Comment by vokaliz** [comment]
Ok! "Прибытие" - this is just "Arrive" in ETA.
